streams/shared/src/main/scala/zio/stream/ZPipeline.scala

```diff
@@ -2078,139 +2078,204 @@ object ZPipeline extends ZPipelinePlatformSpecificConstructors {
    */
   def splitOn(delimiter: => String)(implicit trace: Trace): ZPipeline[Any, Nothing, String, String] =
     ZPipeline.mapChunks[String, Char](_.flatMap(string => Chunk.fromArray(string.toArray))) >>>
-      ZPipeline.splitOnChunk[Char](Chunk.fromArray(delimiter.toArray)) >>>
+      ZPipeline.splitOnChunk[Char, Char](delimiter.toList, true) >>>
```

Comment:
Please use named arguments for booleans like this one
```suggestion
      ZPipeline.splitOnChunk[Char, Char](delimiter.toList, allowEmpty = true) >>>
```